stubs/pymacaroons/pymacaroons/macaroon.pyi

```diff
@@ -0,0 +1,49 @@
+from collections.abc import Collection
+
+from pymacaroons import Caveat
+from pymacaroons.binders import HashSignaturesBinder
+from pymacaroons.caveat_delegates import FirstPartyCaveatDelegate, ThirdPartyCaveatDelegate
+from pymacaroons.serializers import BaseSerializer, JsonSerializer
+
+MACAROON_V1: int
+MACAROON_V2: int
```

Comment:
These should be final, otherwise  the overload trick doesn't work with these constants:

```suggestion
MACAROON_V1: Final = 1
MACAROON_V2: Final = 2
```